wp-admin/js/widgets/media-widgets.js

```diff
@@ -111,7 +111,9 @@ wp.mediaWidgets = ( function( $ ) {
 
 			// Re-render the preview when the attachment changes.
 			control.selectedAttachment = new wp.media.model.Attachment( { id: 0 } );
+			control.renderPreview = _.debounce( control.renderPreview );
```

Comment:
The reason I added the `debounce` is because the preview is also now re-rendering in due to changes to `control.model`. So when you change the media, it will result the `model` being changed (the `attachment_id` and `url` props) as well as the `selectedAttachment` model, resulting in `renderPreview` being called twice. So this was intended to be a minor DOM performance improvement to not waste cycles.